File: test/before_train/trainset_and_testset/java-py/HumanEval_42_incr_list.py
from typing import List

def incr_list(l: List[int]) -> List[int]:
    return l.stream().map(e -> e + 1).collect(Collectors.toCollection(ArrayList::new));

def check(candidate):
    assert candidate([]) == []
    assert candidate([3, 2, 1]) == [4, 3, 2]
    assert candidate([5, 2, 5, 2, 3, 3, 9, 0, 123]) == [6, 3, 6, 3, 4, 4, 10, 1, 124]

def test_check():
    check(incr_list)

test_check()

File: codes/src-tgt/java-cpp/HumanEval_42_incr_list.cpp
### Java version

import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    public static ArrayList<Long> incrList(ArrayList<Long> l) {
        return l.stream().map(e -> e + 1).collect(Collectors.toCollection(ArrayList::new));
    }
    

}


### C++ version

#include<assert.h>
#include<bits/stdc++.h>
#include<vector>
#include<sstream>
#include<regex>
#include<map>
#include<set>
std::vector<long> incr_list(std::vector<long> l) {
  std::vector<long> res;
  for (auto e : l) {
    res.push_back(e + 1);
  }
  return res;
}

